src/common/components/vanilla-modules/forms/form.js

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+import styles from '../../../style/vanilla/css/forms.css';
+
+export default function Form(props) {
+  const { inline=false, stacked=false } = props;
+  const className = classNames({
+    [styles['p-form--inline']]: inline,
+    [styles['p-form--stacked']]: stacked
+  });
+
+  return (
+    <form
+      onSubmit={ props.onSubmit }
+      noValidate={ true }
+      className={ className }
+    >
+      { props.children }
+    </form>
+  );
+}
+
+Form.propTypes = {
+  children: PropTypes.node,
+  onSubmit: PropTypes.func,
+  stacked: PropTypes.bool,
+  inline: PropTypes.bool
```

Comment:
Can a form be both `stacked` and `inline` at the same time?
Having 2 separate props for that kinda suggest that.

If it can't (at least thats how I understand it), maybe it should be a single `appearance` prop with 2 possible values?